.github/actions/upload_test_artifacts/action.yaml

```diff
@@ -38,9 +38,11 @@ runs:
 
         FLAGS=""
         if [[ "${{ matrix.platform }}" == *android* ]]; then
-          FLAGS="--archive-per-target --use-android-deps-path --flatten-deps"
+          FLAGS="--compression gz --archive-per-target --use-android-deps-path --flatten-deps"
```

Comment:
nit: can we add a comment before this if statement explaining why zstd can't be used on these platforms?